ui/src/app/components/take-demographic-survey-v2.tsx

```diff
@@ -4,10 +4,7 @@ import { useLocation } from 'react-router-dom';
 import { NotificationBanner } from 'app/components/notification-banner';
 import { AoU } from 'app/components/text-wrappers';
 import { reactStyles } from 'app/utils';
-import {
-  DEMOGRAPHIC_SURVEY_V2_NOTIFICATION_END_DATE,
```

Comment:
Is it safe get rid of this constant too?

Reply:
I was thinking of opening a new PR for second part of cleanup i,e in signed-in,tsx But now i am adding it in this PR itself